src/components/WebComponentProject/WebComponentProject.jsx

```diff
@@ -82,6 +83,15 @@ const WebComponentProject = ({
     }
   }, [projectIdentifier]);
 
+  renderer.link = function (data) {
```

Comment:
it would be good to get a test in for this, just checking the correct stuff is passed through and a link is rendered